tests/vmi_multus_test.go

```diff
@@ -591,49 +610,49 @@ var _ = Describe("SRIOV", func() {
 		sriovResourceName = "openshift.io/sriov_net"
 	}
 
-	createSriovNetwork := func(name, namespace, resourceName, networkNamespace  string, vlan int, trust, spoofCheck, linkState bool) error {
-		spoofCheckString := func (trust bool) string {
-			if ! spoofCheck {
+	createSriovNetwork := func(name, namespace, resourceName, networkNamespace string, vlan int, trust, spoofCheck, linkState bool) error {
```

Comment:
I think this change relates to this https://github.com/kubevirt/kubevirt/pull/3551/commits/ea26c7c19d29a90bd32d860f4f3ad668fc273084
commit

Reply:
I pushed new commits to address that, now this changes is in the right place.